client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -269,7 +270,7 @@ class CancelPurchaseButton extends Component {
 	submitCancelAndRefundPurchase = () => {
 		const { purchase, selectedSite } = this.props;
 		const refundable = isRefundable( purchase );
-
+		const cancel_bundled_domain = this.props.cancelBundledDomain;
```

Comment:
We should use https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions code in Calypso.